src/main/java/com/autotune/common/data/result/AggregationInfoResult.java

```diff
@@ -15,13 +15,32 @@
  *******************************************************************************/
```

Comment:
This needs to be moved to the `common/data/metrics` dir as well. Also rename it to be consistent with the other classes in the metrics dir

Reply:
Done